### ISSUE TITLE
Most plugins not loading with wine-staging 5.10 in Bitwig Studio 3.2

### ISSUE THREAD

**Issue by siborg** [robbert-vdh/yabridge#19]
I have updated to wine-staging 5.10 yesterday. Looks like the bridged plugins either do not load or if they load, then the UI is not visible. 

Runninng on arch, with standard kernel 5.6.15-1.

Will upload logs as soon as  I have a moment.

**Comment by robbert-vdh** [comment]
I hadn't tried Wine 5.10 yet (Manjaro Stable is still shipping 5.9), but I just installed Wine 5.10 from an Arch mirror and it seems like it does break a lot of things. I'm currently compiling `wine-staging-git` just to make sure that it's not a dependency issue (very unlikely, but you never know). I'm seeing a lot of memory errors and assertion failures just trying to open winecfg. Opening winecfg works when the Wine prefix does not yet exist, but it errors out the same way when running the command a second time. These are the errors I'm getting:

<details>
<summary>When opening winecfg</summary>

```
002c:err:virtual:free_ranges_remove_range range 0x10000000 - 0x10020000 is already partially mapped
s\system32\wineboot.exe: ../../../wine-staging/dlls/ntdll/unix/virtual.c:581: free_ranges_remove_range: Assertion `range->base <= view_base && range->end >= view_end' failed.
0024:err:environ:run_wineboot failed to start wineboot c00000e5
0034:err:virtual:free_ranges_remove_range range 0x10000000 - 0x10020000 is already partially mapped
s\system32\explorer.exe: ../../../wine-staging/dlls/ntdll/unix/virtual.c:581: free_ranges_remove_range: Assertion `range->base <= view_base && range->end >= view_end' failed.
Gtk-Message: 17:10:20.893: Failed to load module "appmenu-gtk-module"
Gtk-Message: 17:10:21.021: Failed to load module "appmenu-gtk-module"
0024:fixme:winediag:start_process Wine Staging 5.10 is a testing version containing experimental patches.
0024:fixme:winediag:start_process Please mention your exact version when filing bug reports on winehq.org.
003c:err:virtual:free_ranges_remove_range range 0x10000000 - 0x100e0000 is already partially mapped
s\system32\winecfg.exe: ../../../wine-staging/dlls/ntdll/unix/virtual.c:581: free_ranges_remove_range: Controletest 'range->base <= view_base && range->end >= view_end' faalt.
0024:err:winecfg:WinMain failed to restart 64-bit L"C:\\windows\\system32\\winecfg.exe", err 1359
0024:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
0024:err:winediag:nodrv_CreateWindow The explorer process failed to start.
```
</details>

Are you seeing similar errors? I have heard reports from people having no issues with Wine Staging 5.10 on Ubuntu, so it might just be a downstream issue.

**Comment by siborg** [comment]
Yes. my errors are similar. even regedit won't start for me with 5.10 but downgrading to 5.9 immediately resolves all issues. I am using wine-starging from multilib repo in Arch.

**Comment by zezic** [comment]
I can add that yabridge works well with Wine Staging 5.10 on **Ubuntu** 20.04.

**Comment by robbert-vdh** [comment]
It must be some kind of downstream issue. I just tried using Wine Staging 5.10 from the official WineHQ repos on Ubuntu and the issue is not present there.

**Comment by siborg** [comment]
Looks like an issue with wine-staging 

https://bugs.winehq.org/show_bug.cgi?id=49326

patch is availabe apparently, just not implemented for Arch yet.

**Comment by robbert-vdh** [comment]
Weird that that slipped! Hopefully Arch will push a new version of Wine Staging 5.10 with this patch because it seems to break everything. Let's just leave this issue open for now in case someone else runs into the same issue!

**Comment by siborg** [comment]
patched wine-staging-5.10-2 is available in arch mutlilib repositories. All working ok with this patched version so far.

**Comment by robbert-vdh** [comment]
Great! I just checked it out myself and everything indeed seem to be back to normal again. Glad they pushed a fix for this.
